packages/components/src/theme/navbar.scss

```diff
@@ -233,6 +233,18 @@ $nav-menu-btn-width: 85px;
   }
 }
 
+@media (min-width: 1200px) {
+    .product-menu-content {
+        width: 1000px;
+    }
+}
+
+@media (min-width: 1450px) {
+    .product-menu-content {
+        width: 1230px;
+    }
+}
+
```

Comment:
See my note in the PR comments. These widths seem fine since they are less then the media width, but because of the positioning of the menu below the header button, they are offset from the left side by a few hundred pixels which looks to result in the right side of the menu popup getting clipped from view.

Reply:
Due to the test outages on the branch also noting this and the fact that the original issue doesn't call out the megamenu width as being a problem, I'm punting the width adjustment to a new issue! Thanks for calling this out Cory 👍

Reply:
That sounds good. Thanks Rosaline. I pulled down the update and verified the menu width is back to normal and the assay column update to stick to a single column and show the "show all" link works as expected.